### ISSUE TITLE
Gather requirements for eCommerce application

### ISSUE THREAD

**Issue by msboffl** [msboffl/Emart#1]

**Comment by NevilleMthw** [comment]
Hi, what issue are you currently facing with your web app?

**Comment by msboffl** [comment]
Hi, i'm not facing any issue, I have started self project, that's why i created  first issue on my project, if you're work with me, please let me know

**Comment by NevilleMthw** [comment]
Yeah it seems good with me, I can assist in anyway I can. Do let me know.

Cheers,
Neville

**Comment by msboffl** [comment]
Hi, Can you please suggest to me which feature we include in our eCommerce project then after we can create issues

**Comment by NevilleMthw** [comment]
For the basic functions of an e-commerce application, would be:

1. Sign up and login.
2. Add, edit and remove from cart.
3. Product/service stock updation, out-of-stock etc.

This would be the basic functionality for the application. Maybe additional stuff could be figured out later.

**Comment by msboffl** [comment]
Ok, got it, I will create the issues that you mentioned above

we can use sass & bootstap also

I

**Comment by NevilleMthw** [comment]
Sounds good, can I know what products/services you want to put on your website?

**Comment by msboffl** [comment]
we can put all products like - electronics, clothes

**Comment by NevilleMthw** [comment]
Seems good, do you want me to create a frontend for products? Like card design?

**Comment by msboffl** [comment]
Card design means..??,  I'm not getting
